pkgs/applications/misc/scli/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, buildPythonApplication, fetchFromGitHub, signal-cli, urwid
+, urwid-readline, dbus }:
+
+buildPythonApplication rec {
+  pname = "scli";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "isamert";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-hWzpqj/sxPq/doxdmytnj5rh2qKQE71WMB0ugomWhHg";
+  };
+
+  propagatedBuildInputs = [ signal-cli urwid urwid-readline dbus ];
+  dontBuild = true;
+
+  checkPhase = ''
+    # scli attempts to write to these directories, make sure they're writeable
+    export XDG_DATA_HOME=$(mktemp -d)
+    export XDG_CONFIG_HOME=$(mktemp -d)
+    ./scli --help > /dev/null # don't spam nix-build log
+    test $? == 0
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m755 -D scli $out/bin/scli
```

Comment:
The shebang at https://github.com/isamert/scli/blob/0c5de11edc8aaa7a9547d92376db0e0b8195836b/scli#L1 needs fixing.

Reply:
fixed in https://github.com/NixOS/nixpkgs/commit/20bb6ae50695c4614492b16b62464fcc8eeb8107